clang/test/Driver/sycl-offload-static-lib-2.cpp

```diff
@@ -34,19 +38,23 @@
 // RUN: touch %t-2.o
 // RUN: touch %t-3.o
 // RUN: %clangxx -target x86_64-unknown-linux-gnu -fsycl %t_lib.a -### %t-1.o %t-2.o %t-3.o 2>&1 \
-// RUN:   | FileCheck %s -check-prefix=STATIC_LIB_MULTI_O
-// STATIC_LIB_MULTI_O: clang-offload-bundler{{.*}} "-type=o" {{.*}} "-inputs={{.+}}-1.o"
-// STATIC_LIB_MULTI_O: clang-offload-bundler{{.*}} "-type=o" {{.*}} "-inputs={{.+}}-2.o"
-// STATIC_LIB_MULTI_O: clang-offload-bundler{{.*}} "-type=o" {{.*}} "-inputs={{.+}}-3.o"
-// STATIC_LIB_MULTI_O: clang-offload-bundler{{.*}} "-type=a" {{.*}} "-outputs=[[OUTFILE:.+\.a]]"
+// RUN:   | FileCheck %s -check-prefix=STATIC_LIB_MULTI_O -DBUNDLE_TRIPLE=sycl-spir64-unknown-unknown-sycldevice
```

Comment:
Treading on the path of https://github.com/intel/llvm/pull/3929:
```suggestion
// RUN:   | FileCheck %s -check-prefix=STATIC_LIB_MULTI_O -DBUNDLE_TRIPLE=sycl-spir64-unknown-unknown
```
There are a couple more occurrences in subsequent test cases.

Reply:
> There are a couple more occurrences in subsequent test cases.

There are many of them!

Reply:
Just fixed them all